data/index.js

```diff
@@ -0,0 +1,5 @@
+import postFunctions from "./posts.js";
+
+const postData = postFunctions;
+
+export default { postData };
```

Comment:
We could go shorthand in this data index and only have a bunch of statements like this:

```suggestion
export {default as postData} from "./posts.js";
```